projects/dxilconv/lib/DxbcConverter/DxbcConverter.cpp

```diff
@@ -7083,7 +7090,7 @@ CompType DxbcConverter::InferOperandType(const D3D10ShaderBinary::CInstruction &
       }
     }
 
-    default: LLVM_FALLTHROUGH;
+    default: break;
```

Comment:
I suppose clang-format will take care of this when the time comes, but it is a nitpicky inconsistency that this is on the same line where the others all created a newline.

Reply:
Fixed.